Widgets/Widgets/VarmintWidgetButton.cs

```diff
@@ -50,6 +52,7 @@ private void Render(GameTime gameTime, VarmintWidget widget)
             if (BackgroundImage != null)
             {
                 LineWidth = 0;
+                BackgroundColor = Color.White;
```

Comment:
We want to avoid setting the background color directly in the code, especially with an image because the image is likely to have transparency.